swift/Podfile.lock

```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: cbcc5bab5b64fa9f7863e62cc106d25d3fc1dfc5
 
-COCOAPODS: 1.7.3
+COCOAPODS: 1.7.1
```

Comment:
Can you upgrade your cocoapods? Or don't commit this file

Reply:
oops, my bad. didn't mean to commit it.